{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestComplexScenario"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "22390-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608373844628, "uuid": "d9102e0d-dcd9-4add-84dd-e438ee575e2c", "historyId": "6df5cbf7c00ce5f14b4c42d69bcd2de9", "name": "test_verification_of_pick_action", "start": 1608373844056, "fullName": "tests.complex_test.TestComplexScenario#test_verification_of_pick_action", "statusDetails": {"message": "RuntimeError: Group 'arm' was not found.", "trace": "self = <tests.complex_test.TestComplexScenario instance at 0x7f25fe00b1b8>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        data_logger('logger/logs/pick_action_start')\n        # Pick from table and place on table\n        result = MoveItPickAndPlace(pick_x = -0.03, pick_y = 0.96 , pick_z = 0.57,\n>                                   place_x = 0.5, place_y = 1.0 , place_z = 0.6)\n\ntests/complex_test.py:173: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/action_client/pick_place_client.py:27: in __init__\n    arm = moveit_commander.MoveGroupCommander(\"arm\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <moveit_commander.move_group.MoveGroupCommander object at 0x7f25fe578090>\nname = 'arm', robot_description = 'robot_description', ns = ''\n\n    def __init__(self, name, robot_description=\"robot_description\", ns=\"\"):\n        \"\"\" Specify the group name for which to construct this commander instance. Throws an exception if there is an initialization error. \"\"\"\n>       self._g = _moveit_move_group_interface.MoveGroupInterface(name, robot_description, ns)\nE       RuntimeError: Group 'arm' was not found.\n\n/opt/ros/kinetic/lib/python2.7/dist-packages/moveit_commander/move_group.py:52: RuntimeError"}, "attachments": [{"source": "88e33eeb-b3bb-43c3-b6d0-1f26ca33822c-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "99a2e27c-9092-487e-b12d-e45a1f638742-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "07364cb3-86b8-4e56-93dd-e12d79f3c3cd-attachment.txt", "type": "text/plain", "name": "stderr"}]}